applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -819,7 +819,22 @@ record_unavailable_greeting(AttachmentName, #mailbox{unavailable_media_id='undef
     MediaId = recording_media_doc(<<"unavailable greeting">>, Box, Call),
     record_unavailable_greeting(AttachmentName, Box#mailbox{unavailable_media_id=MediaId}, Call);
 record_unavailable_greeting(AttachmentName, #mailbox{unavailable_media_id=MediaId}=Box, Call) ->
-    lager:info("recording unavailable greeting  as ~s", [AttachmentName]),
+    case couch_mgr:open_doc(whapps_call:account_db(Call), MediaId) of
```

Comment:
Please use `couch_mgr:open_cache_doc/2` since the resulting JObj should already have been loaded; we can save a trip to the DB in that case.